services/QuillLMS/spec/support/shared_contexts/snapshots/snapshots_top_x_cte.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+RSpec.shared_context 'Snapshots TopX CTE' do
```

Comment:
I think you should delete this context and just include the period context in the test.